installation/linux/ubuntu.md

```diff
@@ -35,53 +41,58 @@ C3C0 A285 34B9 293E AF51  FABD 9F9D DC08 3888 C1CD
 Fluentbit releases (Releases signing key) <releases@fluentbit.io>
 ```
 
-The previous key is still available at [https://packages.fluentbit.io/fluentbit-legacy.key](https://packages.fluentbit.io/fluentbit-legacy.key) and may be required to install previous versions.
+The previous key is [still available](https://packages.fluentbit.io/fluentbit-legacy.key)
+and might be required to install previous versions.
 
 The GPG Key fingerprint of the old key is:
 
 ```text
 F209 D876 2A60 CD49 E680 633B 4FF8 368B 6EA0 722A
 ```
 
-Refer to the [supported platform documentation](../supported-platforms.md) to see which platforms are supported in each release.
+Refer to the [supported platform documentation](../supported-platforms.md) to see
+which platforms are supported in each release.
 
 ## Update your sources lists
 
-On Ubuntu, you need to add our APT server entry to your sources lists, please add the following content at bottom of your **/etc/apt/sources.list** file - ensure to set `CODENAME` to your specific [Ubuntu release name](https://wiki.ubuntu.com/Releases) (e.g. `focal` for Ubuntu 20.04):
+On Ubuntu, you need to add the Fluent Bit APT server entry to your sources lists.
+Add the following content at bottom of your `/etc/apt/sources.list` file. Ensure
+`CODENAME` is set to your specific [Ubuntu release name](https://wiki.ubuntu.com/Releases)
+(for example: `focal` for Ubuntu 20.04):
 
 ```bash
 deb [signed-by=/usr/share/keyrings/fluentbit-keyring.gpg] https://packages.fluentbit.io/ubuntu/${CODENAME} ${CODENAME} main
 ```
 
 ### Update your repositories database
 
-Now let your system update the _apt_ database:
+Now let your system update the `apt` database:
 
 ```bash
 sudo apt-get update
 ```
 
 {% hint style="info" %}
-We recommend upgrading your system (`sudo apt-get upgrade`). This could avoid potential issues with expired certificates.
-{% endhint %}
+Fluent Bit recommends upgrading your system (`sudo apt-get upgrade`). This could
+avoid potential issues with expired certificates.
```

Comment:
```suggestion
Fluent Bit recommends upgrading your system to avoid potential issues
with expired certificates:

`sudo apt-get upgrade`

```

Reply:
Put the command in a proper code fence. I tried to do it as a suggestion, but then GitHub renders the markup and things display all wonky.